compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/programfile/PackageInfo.java

```diff
@@ -132,4 +132,26 @@ public void addAttributeInfo(AttributeInfo.Kind attributeKind, AttributeInfo att
     public AttributeInfo[] getAttributeInfoEntries() {
         return attributeInfoMap.values().toArray(new AttributeInfo[0]);
     }
+
+    /**
+     * Get a deep copy of the packageInfo object.
+     *
+     * @param target packageInfo object to be copied
+     */
+    public void copyTo(PackageInfo target) {
```

Comment:
We dont need this as the original package info can be used

Reply:
Removed this method and used the original package info object with 4e9c501